pyproject.toml

```diff
@@ -11,3 +11,19 @@ ignore = [
     "E731",  # Do not assign a `lambda` expression, use a `def`
 ]
 line-length = 118
+
+[[tool.mypy.overrides]]
+module = "pika.*"
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module = "testmap.*"
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
```

Comment:
meh.  Is this about error messages in vim or about actual fails when linting as part of the static tests run?